.github/workflows/ci_cd_pr.yml

```
@@ -25,7 +25,7 @@ jobs:
     steps:
       - name: Decide which environment should be used
         id: block
-        uses: ansys/actions/check-environment-approval@main
+        uses: ansys/actions/check-environment-approval@feat/check-actions-security-action
```

```suggestion
        uses: ansys/actions/check-environment-approval@main
```
Needs to be reverted before merging